portals/devportal/source/src/app/components/Apis/Details/AsyncApiConsole/AsyncApiUI.jsx

```diff
@@ -62,9 +61,18 @@ export default function AsyncApiUI(props) {
     } = props;
     const { api } = useContext(ApiContext);
```

Comment:
destructure advertiseInfo property to avoid repition

Reply:
simplified here https://github.com/wso2/apim-apps/pull/130/commits/38e365e23c13eb98da349ca0e1f1618291ff9b14